tests/core/offers/test_repository.py

```diff
@@ -167,7 +167,7 @@ def test_returns_offers_filtered_by_imported_creation_mode_when_provided(self):
     @pytest.mark.usefixtures("db_session")
     def should_not_return_event_offers_with_only_deleted_stock_if_filtering_by_time_period(self):
         # given
-        pro = users_factories.UserFactory(isAdmin=True)
+        pro = users_factories.AdminFactory()
```

Comment:
ProFactory() non ?
sinon : 
```suggestion
        admin = users_factories.AdminFactory()
```

Reply:
Bien vu :eyes: